### DISCUSSION TITLE
2.05双击不显示二维码

### DISCUSSION THREAD

**Discussion by moqingzhu1** [messense/aliyundrive-webdav#774] [Q&A] [answered]
手欠升级了一下版本，结果token不能用了，二维码也不显示，一直转圈⭕

**Comment by JashLee0211** [comment, marked as answer]
软路由里三个包都更新了吗，我之前只更了一个，就有这个问题

**Reply by moqingzhu1** [reply]
哪三个包，我以前都是直接更新一个ipk文件，语言包也要更新吗？

**Reply by moqingzhu1** [reply]
> 软路由里三个包都更新了吗，我之前只更了一个，就有这个问题

确实如你所言，后面两个包更新了就好了，感谢
